### PR TITLE
Add `--eval` / `-e` and `--print` / `-p` flags

### LINKED ISSUE #104: Add `--eval` flag
Should be simple, a lot of the logic should be copyable from the REPL

### PR DESCRIPTION
**Pull request by BobVarioa**
Fixes #104 

Adds `--print` and `--eval` flags, similar to Node

### CODE REVIEW

File: runner/index.js [hunk 1]
```diff
@@ -108,7 +109,28 @@ if (['precompile', 'run', 'wasm', 'native', 'c', 'profile', 'debug', 'debug-wasm
 
 globalThis.file = file;
 
-if (!file) {
+let source = "";
+
+let evalIndex = process.argv.indexOf("-e");
+if (evalIndex == -1) evalIndex = process.argv.indexOf("--eval");
```
Comment (CanadaHonk) [inline runner/index.js:115]: nit: use strict ops & single quotes

File: runner/index.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 import fs from 'node:fs';
+import util from 'node:util';
```
Comment (CanadaHonk) [inline runner/index.js:3]: I would prefer to just `console.log`, wdyt?

**Review by CanadaHonk — CHANGES_REQUESTED** [review]

File: runner/index.js (hunk 2 above)
Comment (BobVarioa) [reply runner/index.js:3]: I just copied what the REPL does, but if you'd like I can change that

Comment (CanadaHonk) [reply runner/index.js]: Oh lol sorry, no clue why it does that actually, I'll change repl too later

**Review by CanadaHonk — APPROVED** [review]
Looks good ty!

**Pull request merged**